### ISSUE TITLE
Brushing from coordinate-flipped violin plots

### ISSUE THREAD

**Issue by csoneson** [csoneson/iSEE#105]
With the Allen data as an example:
```
1. Column data plot: Select Y-axis = Core.Type, X-axis: NALIGNED.
2. Reduced dimension plot: Receive brush from Column data plot
3. Brush in Column data plot
```
This gives the right results (supposedly) in the Reduced dimension plot, but changes the Column data plot in a weird way

**Comment by LTLA** [comment]
Fixed in eb5ed690166f319f88efc7b0f8d21bbd92a93091.

**Comment by csoneson** [comment]
Great, closing.
